introspection-engine/connectors/sql-introspection-connector/tests/introspection_command_tests.rs

```diff
@@ -822,6 +822,7 @@ fn foreign_keys_are_preserved_when_generating_data_model_from_a_schema() {
                 indices: vec![],
                 primary_key: None,
                 foreign_keys: vec![ForeignKey {
+                    // what does this mean? the from columns are not targeting a specific to column?
```

Comment:
Since the constraint names are not exposed through the prisma schema/datamodel, I think it doesn't matter here.